content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -76,31 +75,79 @@ To combine an event, click the three dots next to an event and select **+ Combin
 
-- **Time to convert** - The time it took for the user to complete the step events.
 
 You can measure these attributes **across all steps** or between **specific steps**.
 
 {{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_conversion.png" alt="Measure attributes across all steps or specific steps." style="width:60%;" >}}
 
-Use the **filter** selector to filter by various criteria that you define.
+Use the **filter** selector to scope to the various criteria you want to define. These filters are applied to all steps in your funnel.
 
 Next, click a datapoint to **investigate the specific attributes** that might have affected conversion rates, such as page load speed, ease of navigation, or checkout experience.
 
+## Conversion computing metrics
+
+### How Datadog computes conversion metrics
+Consider a funnel with events `A → B → C` and event steps: 
+
+```
+A, A, A, B, C, C
+```
+
+In this case, Datadog counts one conversion. This is because the conversion calculations matches only the first occurence of event **A** and the first occurence of event **C** in the sequence. 
+
+<div class="alert alert-info">
+To further illustrate, if the user performs the event sequence <code>A, A, A, B, C, C, A, B, C...</code>, Datadog counts two conversions. The first conversion completes with the sequence <code>A, A, A, B, C, C</code>, and the second conversion begins with the next occurence of <code>A</code>.
+</div>
+
+The average time between steps is calculated based on the average duration between the first and last step of each conversion. This calculation takes all conversions into account regardless of whether you selected `unique` or `total` for the counts.
+
+
+### Choosing a conversion counting method
+
+When computing your conversion, you can choose between two counting methods: **Unique** or **Total**. 
+
+- **Unique**: Counts conversion only once per session, user, or account. For example, if the user completes the funnel sequence `A → B → C` multiple times within the same session (for example, `A → B → C → A → B → C`), it counts as **one conversion**.
+
+- **Total**: Counts a conversion each time the same session, user or account completes the defined funnel. Using the same example (`A → B → C → A → B → C`), this methods counts **two conversions**.
+
+{{< img src="product_analytics/journeys/funnel_analysis/funnel_analysis_conversion.png" alt="Select a conversion measure, whether Unique or Total, to determine how your session conversions are counted." style="width:80%;" >}}
+
+<div class="alert alert-info">
+The <strong>Total</strong> setting doesn't multiply conversions for repeated steps within a single flow, instead it counts complete flows, not the number of times an intermediate step is repeated. 
+
+<br>
+<br
+
+If you switch to <strong>Unique</strong>, then only the first conversion per session (or per user, depending on your analysis scope) is counted.
+</div>
```

Comment:
This information should be incorporated into where you explained what both of these things mean above. Otherwise, a user might miss this and miss key information. When you put an explanation all in one place, that's far less likely to happen!

(Also, a note about structure, even though it won't apply in the final product here - if you explain "unique," followed by "total," in one part of your documentation, you should refer to them in the same order at other times. This helps customers structure information in their heads! It's called parallelism.)